common/game_rules/zz_hfe_vanilla_replace_game_rules.txt

```diff
@@ -21,33 +21,33 @@
 # 		}
 # 	}
 # }
-
+#
 #This rule is a condition for bombarding planets
 #Root = fleet, potential attacker
 #From = planet, potential target
-can_orbital_bombard = {
-	exists = FROM.OWNER
-	NOT = {
-		any_owned_ship = {
-			is_ship_size = colossus
-		}
-	}
-	# mod
-	exists = ROOT.OWNER
-	exists = FROM.CONTROLLER
-	or = {
-		ROOT.OWNER = { is_hostile = FROM.CONTROLLER }
-		AND = {
-			ROOT.OWNER = {
-				NOR = {
-					has_policy_flag = interference_passive
-					has_policy_flag = interference_active
-				}
-			}
-			FROM.OWNER = { is_country_type = primitive }
-		}
-	}
-}
+#can_orbital_bombard = {
```

Comment:
Why removed?

Reply:
Country Types can handle Orbital Bombardment now. The "ORBITAL_BOMBARDMENT = YES" command governs that.

Reply:
I guess pirates will not be able to bombard default or primitive.

Reply:
Pirates are set to have Orbital Bombardment.

Reply:
![image](https://user-images.githubusercontent.com/17663470/82472213-d4d8bd00-9a95-11ea-9171-a3bf8432f075.png)

Reply:
!https://steamuserimages-a.akamaihd.net/ugc/1008189355407098062/1000E6564C231F25B68B870E9CF81F00A372A691/!

Reply:
Okay, I ran the test. Pirates cannot bombard. I seriously did not expect Paradox to be that stupid with their Game Rules. I changed it back.